appinventor/components-ios/src/LocationSensor.swift

```diff
@@ -255,9 +258,10 @@ open class LocationSensor: NonvisibleComponent, CLLocationManagerDelegate {
             // do something here too --> Placemark was nil
             return
           }
-          let postalAddress = CNMutablePostalAddress(placemark: placemark)
-          let addressStr = CNPostalAddressFormatter().string(from: postalAddress)
-          address = addressStr.isEmpty ? address : addressStr
+          if let postalAddress = placemark.postalAddress {
+            let addressStr = CNPostalAddressFormatter().string(from: postalAddress)
+            address = addressStr.isEmpty ? address : addressStr
+          }
```

Comment:
the formatting doesn't seem correct. I wonder if we have to set the ISO code and parts of the address explicitly for EU format?